controlpanel/api/migrations/0006_user_last_name_length.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.7 on 2019-03-27 14:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0005_s3bucket_url'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='user',
+            name='last_name',
+            field=models.CharField(blank=True, max_length=150, verbose_name='last name'),
```

Comment:
This new migration seems to be a left-over from an old change which was never committed.

### [ ⏩ Next ]()

Reply:
I'm a noob but does this also needs `replaces = [('control_panel_api', '0006_user_last_name_length')]`?

Reply:
No, because there is no existing `0006_user_last_name_length` migration to replace.